core/server/services/slack.js

```diff
@@ -42,6 +42,7 @@ function ping(post) {
 
     // Quit here if slack integration is not activated
     if (slackSettings && slackSettings.url && slackSettings.url !== '') {
+        slackSettings.username = slackSettings.username ? slackSettings.username : 'Ghost';
```

Comment:
I don't *think* there should be any side effects to defaulting the option here, but I did this to avoid two separate places where we default to 'Ghost'.